src/python/interpret/api/templates.py

```diff
@@ -39,10 +39,10 @@ def data(self, key=None):
         return self._internal_obj["specific"][key]
 
     def visualize(self, key=None):
-        from ..visual.plot import plot_line, plot_bar, plot_horizontal_bar
-        from ..visual.plot import sort_take, plot_pairwise_heatmap
-
+        from ..visual.plot import plot_line, plot_bar, plot_horizontal_bar, plot_horizontal_bar2
+        from ..visual.plot import get_sort_indexes, sort_take, sort_take2, plot_pairwise_heatmap
         data_dict = self.data(key)
+        mli_data = self.data(-1)["mli"]
```

Comment:
keep string literals in a constants class ?

Reply:
is `-1` a special key for things like mli?

Reply:
So far this is just following current practices. If we want the codebase constantified, that should be a different PR